src/zope/tal/tests/test_talinterpreter.py

```diff
@@ -75,9 +68,13 @@ def test_version_error(self):
         self.macro[0] = ("version", "duh")
 
 
-class _MacroFunkyErrorTest(TestCaseBase):
+class TestMacroFunkyError(TestCaseBase):
 
+    @unittest.expectedFailure
     def test_div_in_p_using_macro(self):
+        # We have not found a solution for this
+        # and it is a deep and undocumented HTML parser issue.
+        # Fred is looking into this.
```

Comment:
Funny this message was there since 24a6868 (this was nearly 14 years ago). Maybe @freddrake is the one mentioned.